akka-docs/src/test/scala/typed/tutorial_3/Device.scala

```diff
@@ -30,21 +33,26 @@ object Device {
   //#write-protocol
 }
 
-class Device(context: ActorContext[Device.Command], groupId: String, deviceId: String)
+class Device(
+    context: ActorContext[Device.Command],
+    groupId: String,
+    deviceId: String,
+    lastTemperatureReading: Option[Double])
     extends AbstractBehavior[Device.Command](context) {
   import Device._
 
-  var lastTemperatureReading: Option[Double] = None
```

Comment:
as Johan said, there are two styles, this is the oo style since it `extends AbstractBehavior` and then a `var` is perfectly fine